### ISSUE TITLE
Farm Second Tab line 3 not work correctly

### ISSUE THREAD

**Issue by LinuxDonald** [unknown-horizons/unknown-horizons#2373]
Uncomment line 195 ind game.sql and click on the second Tab on the Farm building.
Fot that you need TIER.CITIZENS.

@stubb

**Comment by LinuxDonald** [comment]
![2016-03-15 20-01-55 578609](https://cloud.githubusercontent.com/assets/1757651/13790582/7de9438a-eae9-11e5-9754-2795af968a83.png)

**Comment by MasterofJOKers** [comment]
FIFE doesn't seem to support automatic linebreaks in `HBox`. We could remove the grouping by one line per TIER and hard-code 4 entries per line. (We could also keep the grouping by TIER and hard-code 4 entries per line. This would mean 3 in the first line 1 in the last line instead of 4 in all three lines.) Imho it makes sense to remove the grouping as there is no indicator, that any grouping is applied (e.g. a heading for the TIER) and empty lines i.e. TIERs without related buildings are skipped.

@LinuxDonald You wanted to have the diff:
```diff
diff --git a/content/game.sql b/content/game.sql
index 8d0be8f..9a76fa2 100644
--- a/content/game.sql
+++ b/content/game.sql
@@ -198,7 +198,7 @@ INSERT INTO "related_buildings" VALUES(20, 49, 1);
 INSERT INTO "related_buildings" VALUES(20, 60, 1);
 INSERT INTO "related_buildings" VALUES(20, 61, 1);
 INSERT INTO "related_buildings" VALUES(20, 62, 1);
---INSERT INTO "related_buildings" VALUES(20, 69, 1); --out of space
+INSERT INTO "related_buildings" VALUES(20, 69, 1); --out of space
 INSERT INTO "related_buildings" VALUES(45,  3, 0);
 
 CREATE TABLE "tile_set" (
diff --git a/horizons/gui/tabs/buildrelatedtab.py b/horizons/gui/tabs/buildrelatedtab.py
index e465014..48e9298 100644
--- a/horizons/gui/tabs/buildrelatedtab.py
+++ b/horizons/gui/tabs/buildrelatedtab.py
@@ -56,11 +56,11 @@ class BuildRelatedTab(OverviewTab):
                sorted_ids = sorted([(b, Entities.buildings[b].settler_level) for b in building_ids], key=lambda x : x[1])
                container = self.__get_new_container()
                self.current_row = min(building[1] for building in sorted_ids)
-               for building_id, level in sorted_ids:
+               for i, (building_id, level) in enumerate(sorted_ids):
                        if level <= self.instance.owner.settler_level: # available in build menu?
                                button = self._create_build_buttons(building_id, container)
-                               # check whether to start new line (for new tier row)
-                               if level > self.current_row:
+                               # check whether to start new line (only 4 fit per line, no auto-newline supported)
+                               if i and i % 4 == 0:
                                        self.current_row = level
                                        parent_container.addChild(container)
                                        container = self.__get_new_container()
```

**Comment by LinuxDonald** [comment]
I found an other bug when you change line: 187 , 194, 202 in game.sql from 0 to 1
you will become this problem the last icon is cuted:
![2016-08-21 22-42-20 814217](https://cloud.githubusercontent.com/assets/1757651/17839712/b6d95944-67f0-11e6-8070-8bf651f78b6b.png)

**Comment by MasterofJOKers** [comment]
That should be fixable by changing [this line](https://github.com/unknown-horizons/unknown-horizons/blob/master/content/gui/xml/ingame/buildmenu/related_buildings.xml#L15). Give more vertical space (height, second attribute to size) to the VBox.

**Comment by LinuxDonald** [comment]
You are the best @MasterofJOKers :) could you please create an final patch please :)

**Comment by LinuxDonald** [comment]
i changed it to 210. I think its engouh for now. We dont have more farms for now.

**Comment by MasterofJOKers** [comment]
This should be fixed with 99e4d0d3.
